solvebio/cli/data.py

```diff
@@ -268,3 +302,69 @@ def import_file(args):
         mesh_url = 'https://my.solvebio.com/activity/'
         print("Your import has been submitted, view details at: {0}"
               .format(mesh_url))
+
+
+def apply_tags(object_, tags, dry_run=False):
+
+    def lowercase(x):
+        return x.lower()
+
+    existing_tags = map(lowercase, object_.tags)
+    new_tags = [tag for tag in tags if lowercase(tag) not in existing_tags]
```

Comment:
I think that this logic (tag case) is already handled by the API

Reply:
It is, however I didn't want to make an API request to the object if there is no change to be made.

Reply:
Up to you

Reply:
ok ill leave it